src/lib.rs

```diff
@@ -197,17 +220,17 @@ impl Watch {
 
         let metadata = metadata();
 
-        self.exclude_path(metadata.target_directory.as_std_path());
+        self.exclude_path(&metadata.target_directory);
 
         if self.watch_paths.is_empty() {
-            log::trace!("Watching {}", &metadata.workspace_root);
+            log::trace!("Watch: Watching {}", &metadata.workspace_root);
             watcher
                 .watch(&metadata.workspace_root, RecursiveMode::Recursive)
                 .context("cannot watch this crate")?;
         } else {
             for path in &self.watch_paths {
                 match watcher.watch(&path, RecursiveMode::Recursive) {
-                    Ok(()) => log::trace!("Watching {}", path.display()),
+                    Ok(()) => log::trace!("Watch: Watching {}", path.display()),
```

Comment:
Redundant prefix again.

I guess you meant to classify the logs per struct but this is not a useful information and it's annoying to maintain. Imagine you have to rename it.